nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/groovy/org/apache/nifi/controller/repository/crypto/EncryptedFileSystemRepositoryTest.groovy

```diff
@@ -64,9 +61,9 @@ class EncryptedFileSystemRepositoryTest {
             (NiFiProperties.CONTENT_REPOSITORY_ENCRYPTION_KEY_PROVIDER_LOCATION)            : ""
     ]
 
-    @BeforeClass
+    @BeforeAll
     static void setUpOnce() throws Exception {
-        Assume.assumeTrue("Test only runs on *nix", !SystemUtils.IS_OS_WINDOWS)
+        Assumptions.assumeTrue(!SystemUtils.IS_OS_WINDOWS, "Test only runs on *nix")
```

Comment:
This class and others should be changed to use the new `DisabledOnOs(OS.WINDOWS)` JUnit 5 annotation, will clean up some of these `SystemUtils` and `BeforeAll` setup steps.

Reply:
@exceptionfactory I made the changes to this class and TestFileSystemRepository. I noticed that EncryptedSequentialAccessWriteAheadLogTest has this also but it has not been converted over to Junit 5. Do you mind if make the changes for JUnit 5 on this class also?

Reply:
Sure, feel free to make the additional changes.

Reply:
So I made the changes to EncryptedSequentialAccessWriteAheadLogTest but I am not sure what to do as the 2 tests are run twice (I am not sure why). The first time they both pass and the second time they both fail. This happens in Intellij and with Maven. I am attaching the stacktrace from running it in Maven. 
[EncryptedSequentialAccessWriteAheadLogTest_stacktrace.txt](https://github.com/apache/nifi/files/10212804/EncryptedSequentialAccessWriteAheadLogTest_stacktrace.txt)

Reply:
From the stack trace, it appears to be using an older JUnit Runner. Make sure to remove the `extends GroovyTestCase` from the class as part of the changes.

Reply:
This may be the cause [Why does JUnit run tests twice with different results](https://stacko(https://stackoverflow.com/questions/50751861/why-does-junit-run-tests-twice-with-different-resuls)

Reply:
@exceptionfactory That was it. Thanks! I just had to import all the Assertions in assertions to cover the ones defined in GroovyTestCase